src/EditorFeatures/Test/CodeFixes/CodeFixServiceTests.cs

```diff
@@ -1171,7 +1171,7 @@ void M()
             static async Task VerifyCachedDiagnosticsAsync(Document sourceDocument, bool expectedCachedDiagnostic, TextSpan testSpan, DiagnosticIncrementalAnalyzer diagnosticIncrementalAnalyzer)
             {
                 var cachedDiagnostics = await diagnosticIncrementalAnalyzer.GetCachedDiagnosticsAsync(sourceDocument.Project.Solution, sourceDocument.Project.Id, sourceDocument.Id,
-                    includeSuppressedDiagnostics: false, includeNonLocalDocumentDiagnostics: true, CancellationToken.None);
+                    includeSuppressedDiagnostics: false, includeLocalDocumentDiagnostics: true, includeNonLocalDocumentDiagnostics: true, CancellationToken.None);
```

Comment:
Note: I needed to add this new flag `includeLocalDocumentDiagnostics` to `GetCachedDiagnosticsAsync` API to ensure that our CodeAnalysis based ProjectDiagnosticSource can request for only no-location diagnostics by explicitly specifying `includeLocalDocumentDiagnostics: false` in the call.